### PR TITLE
Artp 302/blotter notification flash

### PR DESCRIPTION
**Pull request by pdeleastar**

### CODE REVIEW

File: src/client/src/routes/NotificationRoute.tsx [hunk 1]
```diff
@@ -14,27 +13,53 @@ interface State {
...
+        <TradeNotification
+          message={message.tradeNotification}
+          dismissNotification={this.onDismissNotification}
+          highlightTradeInBlotter={this.highlightTradeInBlotter}
+        />
+      </ThemeProvider>
+    ) : (
       <ThemeProvider>
-        <TradeNotification message={message.tradeNotification} dismissNotification={this.onDismissNotification} />
+        <TradeNotification message={debugTrade} dismissNotification={() => console.log('Dismiss notification')} />
```
Comment (danmcdaid) [inline src/client/src/routes/NotificationRoute.tsx]: console.log

File: src/client/src/rt-components/platform/adapters/openfin/openFin.ts [hunk 2]
```diff
@@ -73,7 +75,7 @@ export default class OpenFin implements PlatformAdapter {
       new fin.desktop.Notification({
         url: '/notification',
         message,
-        duration: 20000,
-      }),
+        timeout: 20000,
+      } as any),
```
Comment (snony) [inline src/client/src/rt-components/platform/adapters/openfin/openFin.ts]: Is it possible to avoid the "any" type?

Comment (keshaldesai) [reply src/client/src/rt-components/platform/adapters/openfin/openFin.ts]: Not until the types are fixed

File: src/client/src/ui/blotter/epics/blotterHighlightEpic.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      const tradeNotification = message[0].tradeNotification
+      const trade = switchHighlight(state.blotterService.trades[tradeNotification.tradeId])
+      return highlightTradeAction({ trades: [trade] })
+    }),
+  )
+}
+
+const removeHighlightTradeEpic: ApplicationEpic = (action$, state$, { platform }) => {
+  return action$.pipe(
+    ofType<Action, any>(BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE),
```
Comment (keshaldesai) [inline src/client/src/ui/blotter/epics/blotterHighlightEpic.ts]: I think the second type here is `BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE`

File: src/client/src/ui/blotter/actions.ts [hunk 4]
```diff
@@ -3,14 +3,22 @@ import { TradesUpdate } from './blotterService'
...
+  BLOTTER_SERVICE_REMOVE_HIGHLIGHT_TRADE = '@ReactiveTraderCloud/BLOTTER_SERVICE_REMOVE_HIGHLIGHT_TRADE',
 }
 
 export const BlotterActions = {
   createNewTradesAction: action<BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_NEW_TRADES, TradesUpdate>(
-    BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_NEW_TRADES
+    BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_NEW_TRADES,
+  ),
+  subscribeToBlotterAction: action(BLOTTER_ACTION_TYPES.SUBSCRIBE_TO_BLOTTER_ACTION),
+  highlightTradeAction: action<BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE, any>(
```
Comment (keshaldesai) [inline src/client/src/ui/blotter/actions.ts]: type

File: src/client/src/ui/blotter/actions.ts [hunk 5]
```diff
@@ -3,14 +3,22 @@ import { TradesUpdate } from './blotterService'
...
+    BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE,
+  ),
+  removeHighlightTradeAction: action<BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_REMOVE_HIGHLIGHT_TRADE, any>(
```
Comment (keshaldesai) [inline src/client/src/ui/blotter/actions.ts]: type

File: src/client/src/ui/blotter/epics/blotterHighlightEpic.ts [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+const switchHighlight = (trade: Trade) => ({
+  ...trade,
+  highlight: !trade.highlight,
+})
+
+const highlightTradeEpic: ApplicationEpic = (action$, state$, { platform }) => {
+  const interopObservable$ = fromEventPattern(
+    (handler: any) => platform.interop!.subscribe('*', 'highlight-blotter', handler),
+    (handler: any) => platform.interop!.unsubscribe('*', 'highlight-blotter', handler),
```
Comment (keshaldesai) [inline src/client/src/ui/blotter/epics/blotterHighlightEpic.ts]: See if you can type this handler

**Pull request merged**